jobspec2/parse_test.go

```diff
@@ -0,0 +1,145 @@
+package jobspec2
+
+import (
+	"io/ioutil"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/hashicorp/nomad/jobspec"
+	"github.com/stretchr/testify/require"
+)
+
+func TestEquivalentToHCL1(t *testing.T) {
```

Comment:
As a bonus, this is a nice way of saving us from writing a whole lot of test code!